assets/js/google-tag.js

```diff
@@ -0,0 +1,15 @@
+---
+layout: blank
+---
+
+window.dataLayer = window.dataLayer || [];
+function gtag(){window.dataLayer.push(arguments);}
```

Comment:
`argument` seems to never be defined - perhaps you missed it out as `gtag(arguments)`?

Reply:
This is actually the given snippet from GA - see [here](https://support.google.com/analytics/answer/10271001?hl=en#analyticsjs-enable-basic&zippy=%2Cin-this-article)

Reply:
:monkas: how does `arguments` get in scope unless it's a global variable :wutface: